### PR TITLE
fix: User avatar not loading in EditProfileFragment

### LINKED ISSUE #653: EditProfileFragment does not display the user's avatar picture
**Describe the bug**
The EditProfileFragment does not display the user's profile avatar correctly. Instead, it only displays a hardcoded placeholder image.

**To Reproduce**
1. Go to profile tab > menu > Edit Profile
2. Update image avatar
3. Repeat step 1
4. The fragment still displays the placeholder image

**Expected behavior**
1. Go to profile tab > menu > Edit Profile
2. Update the image avatar
3. Repeat step 1
4. The fragment should display the updated profile image, and not the placeholder image.

**Screenshots**
</img> 
</img> 

**Smartphone Info:**
Google Pixel
Pie 9.0

**Would you like to work on the issue?**
- [x] Yes

### PR DESCRIPTION
**Pull request by haroldadmin**
Fixes #653 
Changes: Added logic to load the user's profile avatar in the EditProfileFragment.

Screenshots for the change:
fixed-screenshot

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Please follow PR guidelines

**Comment by haroldadmin** [comment]
> Please follow PR guidelines

Hello! This is my first PR on any FOSS project. I'd really appreciate some guidance on what I'm doing wrong. Please let me know the PR guidelines.

**Comment by iamareebjamal** [comment]
https://github.com/fossasia/open-event-android/blob/development/README.md#code-practices

File: app/src/main/java/org/fossasia/openevent/general/auth/EditProfileFragment.kt [hunk 1]
```diff
@@ -52,8 +52,14 @@ class EditProfileFragment : Fragment() {
             it?.let {
                 val userFirstName = it.firstName.nullToEmpty()
                 val userLastName = it.lastName.nullToEmpty()
+                val imageUrl = it.avatarUrl.nullToEmpty()
                 rootView.firstName.setText(userFirstName)
                 rootView.lastName.setText(userLastName)
+                Picasso.get()
+                    .load(imageUrl)
+                    .placeholder(AppCompatResources.getDrawable(context!!, R.drawable.ic_account_circle_grey_24dp)!!)
```
Comment (nikit19) [inline app/src/main/java/org/fossasia/openevent/general/auth/EditProfileFragment.kt]: Can you remove the exclamation marks and click on the red bulb that appears. 
You can then select the option to wrap it with ?. let call .

Comment (haroldadmin) [reply app/src/main/java/org/fossasia/openevent/general/auth/EditProfileFragment.kt]: Yes, I'm adding null checks to it right now. Thank you!

Comment (haroldadmin) [reply app/src/main/java/org/fossasia/openevent/general/auth/EditProfileFragment.kt]: I've created a new commit that adds a null check to the drawable and context. Is this okay?

**Comment by codepoet2017390** [comment]
Yes the project maintainers will take care of that further

**Comment by nikit19** [comment]
Looks good code wise. 
We follow this guide for writing proper commit messages. https://chris.beams.io/posts/git-commit/
Could update the PR title and commit message accordingly

**Comment by haroldadmin** [comment]
@nikit19 I have amended the commit title and the PR title accordingly. Thanks for the feedback 😊

**Comment by haroldadmin** [comment]
@nikit19 I have amended the commit title and the PR title accordingly. Thanks for the feedback 😊

**Comment by haroldadmin** [comment]
So the PR is now accepted and will be merged? Woohoo! Thanks for the help @codepoet2017390 @iamareebjamal @nikit19 😊

This was my first pull request to an open source project. Looking forward to contributing more!

**Pull request merged**